src/Board.hs

```diff
@@ -4,7 +4,9 @@ module Board (
     isStartingLayout,
     Dimensions(..),
     Board(..),
-    SlideError(..)
+    SlideError(..),
+    Width,
+    Height
```

Comment:
Oh!  Width and Height!  I had added those later for clarity and forgot to propagate elsewhere.  Thank you for catching that.  Is there a way to convince the compiler to be picky about `type` usage?  One option is type constructors, but I initially meant these to help with readability.